modules/python/test/test_imread.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+'''
+Test for imread
+'''
+
+# Python 2/3 compatibility
+from __future__ import print_function
+
+import cv2 as cv
+import numpy as np
+import sys
+
+from tests_common import NewOpenCVTests
+
+class dft_test(NewOpenCVTests):
+    def test_dft(self):
+
+        img = np.zeros((1000, 1000), dtype=np.uint8)
+        subImg = img[64:576, 128:640]
+        subImg = cv.imread(self.find_file("cv/shared/lena.png"), subImg, cv.IMREAD_GRAYSCALE)
```

Comment:
Can it work without assignment?
```python
cv.imread(self.find_file("cv/shared/lena.png"), subImg, cv.IMREAD_GRAYSCALE)
```

Reply:
sorry i don't understand what you mean

Reply:
Can this method work without an assignment?
replace
```python
subImg = cv.imread(self.find_file("cv/shared/lena.png"), subImg)
```
to
```python
cv.imread(self.find_file("cv/shared/lena.png"), subImg)
```